### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Name property is empty for document in "Service Principal Creation Status dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1389]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , select the required subscriptions and select "Start" button.
6. In "Service Principal Creation Status dialog", tab to document and observe the name property using inspect.

**Actual Result:**
Name property is empty for the document..

**Narrator behaviour:**
Tab

**NVDA behaviour:**
edit  read only
blank

**Expected Result:**
Name property for the document should be "Service Principal Creation Status".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![service principal creatation status-document-no name](https://user-images.githubusercontent.com/22956130/35682454-c7eb0db8-0786-11e8-8250-09d8d2591a57.jpg)
![service principal creatation status-document-nvda](https://user-images.githubusercontent.com/22956130/35682459-cb24b31c-0786-11e8-9b14-01f49ab9dfdb.jpg)

**Comment by Eskibear** [comment]
shipped in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V3.6.0.
Issue still exists
![1389-not fixed](https://user-images.githubusercontent.com/22956130/39797070-cea9071c-5377-11e8-9e22-c169ce020279.jpg)

**Comment by Eskibear** [comment]
tracked in https://bugs.eclipse.org/bugs/show_bug.cgi?id=533976

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. Very low usage based on telemetry
* **Quality of experience**: not impacted. Users can still see the property in the `Value` field
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

The same with #1380

**Comment by jdneo** [comment]
Can show the name property on my side.

Eclipse Version: Photon Release (4.8.0)
Plugin Version: 3.11.0

![image](https://user-images.githubusercontent.com/6193897/47150673-debe9b00-d309-11e8-896b-6774386b32ab.png)

**Comment by Tanushagottipati** [comment]
This issue is still reproducing in the current test pass

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749848
